Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -20,6 +20,24 @@ class ContentBlockerHelper {
 
     static var blockImagesRule: WKContentRuleList?
 
+    enum TrackingProtectionUserOverride {
+        case disallowUserOverride // Option is not offered if tracking protection is off in prefs
+        case allowedButNotSet
+        case forceEnabled
+        case forceDisabled
+    }
+
+    fileprivate var prefOverrideTrackingProtectionEnabled: Bool?
+    var userOverrideForTrackingProtection: TrackingProtectionUserOverride {
+        if !trackingProtectionEnabledInSettings {
+            return .disallowUserOverride
```

Comment:
I like the idea of having long-press to "Reload with TP" even if TP is off in the app settings.
For this patch I didn't do that. I am filing a follow up bug to propose that.

Reply:
This is my only feedback for this PR.

Reply:
thanks, filed for follow up https://bugzilla.mozilla.org/show_bug.cgi?id=1409090#c10